packages/release/media-cdrom.mount

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=CDROM mount(/media/cdrom)
+Requires=dev-cdrom.device
+After=dev-cdrom.device
```

Comment:
It's worth adding a brief explanation of *why* these (`Requires`, `After`, & `WantedBy`) have to be written here, where they otherwise don't usually need to be. This is a slightly odd unit to begin with :)

Reply:
I would expect this mount unit to have the usual set of mount dependencies:
```
DefaultDependencies=no
Conflicts=umount.target
Before=local-fs.target umount.target
```